src/serializers/mod.rs

```diff
@@ -202,13 +203,14 @@ pub fn to_json(
 
 #[allow(clippy::too_many_arguments)]
 #[pyfunction]
-#[pyo3(signature = (value, *, include = None, exclude = None, exclude_none = false, round_trip = false,
+#[pyo3(signature = (value, *, include = None, exclude = None, by_alias = None, exclude_none = false, round_trip = false,
```

Comment:
It seemed a bit weird to me that the python type-hints say `by_alias: bool = True` but the rust signature is `Option<bool>` and even the annotated signature has a default of `None` and not `True`. I'm guessing this is an attempt at representing default values in rust (i.e., `None` means do whatever the default is), but it doesn't seem to me it really achieves any of the benefits:
* Since you have to manually specify `None` if you were to call the function from rust, you don't get the benefit of fewer arguments to put in the signature
* While you can always signal that you explicitly mean to use the "default' by using `None`, considering that if you wanted to change the default value on the python side you'd still have to make sure `None` was interpreted correctly on the rust side, it seems to me that this doesn't really simplify much in terms of maintaining the behavior. 
* The one exception to the previous point I can think of is that if you had to make sure to use the correct default value at all of the call-sites, I could see that being problematic. But considering I don't think we actually call the python methods/functions anywhere from rust code, this too seems unnecessary.

Anyway, not a big deal, sort of a curiosity, but I figured if there _is_ a particularly good reason to do it like this, it would probably be good for me to understand it (since I would otherwise do by_alias = True in the signature here and annotate as having type `bool` in the rust signature).

Reply:
I wasn't using pyo3's `signature` before, see #534 for a full solution option.